stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -37,7 +38,7 @@
     long iteration = 0;
 
     public AbstractWorkerTask() {
-        this.selector = createOperationSelector();
+        this.selector = createOperationSelectorBuilder().build();
```

Comment:
I can give that a try. That would solve the problem that we create several builders at the moment, which is technically not necessary.